app/views/RoomView/List/components/index.ts

```diff
@@ -1,4 +1,3 @@
 export * from './NavBottomFAB';
-export * from './RefreshControl';
```

Comment:
I think we can keep the component, just remove the implementation, who knows, maybe in the future we will reuse it and reimplement this functionality.